src/kotlin_script.cpp

```diff
@@ -240,15 +240,23 @@ void KotlinScript::_update_exports(PlaceHolderScriptInstance* placeholder) {
         KotlinInstance* script_instance {
           dynamic_cast<KotlinInstance*>(_instance_create({}, 0, tmp_object))};
 
-        List<PropertyInfo> properties;
-        get_script_property_list(&properties);
-        for (int i = 0; i < properties.size(); ++i) {
+        List<PropertyInfo> all_properties;
+        get_script_property_list(&all_properties);
+
+        List<PropertyInfo> exported_properties;
+        for (const PropertyInfo& property_info : all_properties) {
+            if (property_info.usage != PropertyUsageFlags::PROPERTY_USAGE_NO_EDITOR) {
```

Comment:
I think this should be `!(property_info.usage & PropertyUsageFlags::PROPERTY_USAGE_NO_EDITOR)`

Reply:
Actually it should be `property_info.usage & PropertyUsageFlags::PROPERTY_USAGE_EDITOR` as `PROPERTY_USAGE_DEFAULT` is defined as `PROPERTY_USAGE_DEFAULT = PROPERTY_USAGE_STORAGE | PROPERTY_USAGE_EDITOR` and `PROPERTY_USAGE_NO_EDITOR` as `PROPERTY_USAGE_NO_EDITOR = PROPERTY_USAGE_STORAGE`